visual-studio-code/spoon.me

```diff
@@ -0,0 +1,60 @@
+#
+# Visual Studio Code spoon.me file
+# https://github.com/spoonium/spoonme/tree/master/visual-studio-code
+#
+# Licensed under the Apache License, Version 2.0
+# http://www.apache.org/licenses/LICENSE-2.0
+
+layer spoonbrew/clean
+
+########################################
+# Define meta information
+########################################
+
+meta title = "Visual Studio Code"
+meta namespace = "microsoft"
+meta name = "visual-studio-code"
+
+var tag = "0.5.0"
```

Comment:
couple suggestions:

- can we begin using "uses" now up top?
- suggest we use just "clean" rather than the fully qualified



On Aug 7, 2015, at 9:10 PM, Victor Chugunov <notifications@github.com<mailto:notifications@github.com>> wrote:


In visual-studio-code/spoon.me<https://github.com/spoonapps/spoonme/pull/53#discussion_r36551294>:

> +# https://github.com/spoonium/spoonme/tree/master/visual-studio-code
> +#
> +# Licensed under the Apache License, Version 2.0
> +# http://www.apache.org/licenses/LICENSE-2.0
> +
> +layer spoonbrew/clean
> +
> +########################################
> +# Define meta information
> +########################################
> +
> +meta title = "Visual Studio Code"
> +meta namespace = "microsoft"
> +meta name = "visual-studio-code"
> +
> +var tag = "0.5.0"


yes, it does actually work.

—
Reply to this email directly or view it on GitHub<https://github.com/spoonapps/spoonme/pull/53/files#r36551294>.